tools/sklearn/nn_classifier.xml

```diff
@@ -104,42 +107,60 @@ pickle.dump(classifier_object,open("$outfile_fit", 'w+'))
 
     <tests>
         <test>
-            <param name="infile_train" value="train_set.tabular" ftype="tabular"/>
+            <param name="infile1" value="train_set.tabular" ftype="tabular"/>
+            <param name="infile2" value="train_set.tabular" ftype="tabular"/>
+            <param name="header1" value="True"/>
+            <param name="header2" value="True"/>
+            <param name="col1" value="1,2,3,4"/>
+            <param name="col2" value="5"/>
             <param name="selected_task" value="train"/>
             <param name="selected_algorithm" value="nneighbors"/>
             <param name="sampling_method" value="KNeighborsClassifier" />
             <param name="algorithm" value="brute" />
             <output name="outfile_fit" file="nn_model01.txt"/>
         </test>
         <test>
-            <param name="infile_train" value="train_set.tabular" ftype="tabular"/>
+            <param name="infile1" value="train_set.tabular" ftype="tabular"/>
```

Comment:
should this also be in a macro?

Reply:
Good point. You want me to change it?

Reply:
Not needed urgently or needed, but if we are at it we could change it at some point.

Reply:
Applying header/column selection to these train/predict tools doesn't seem to improve much, but looks bulky. Do you really like it?

Reply:
No ;) easier userinterface always wins.

Reply:
I also feel the gain/overhead ratio for this PR is low. Feel free to revoke the changes when they bother you.